src/overlays/actors/ovl_En_GeldB/z_en_geldb.c

```diff
@@ -4,24 +4,88 @@
 
 #define THIS ((EnGeldB*)thisx)
 
+typedef enum {
+    /*  0 */ GELDB_WAIT,
+    /*  1 */ GELDB_DEFEAT,
+    /*  2 */ GELDB_DAMAGED,
+    /*  3 */ GELDB_JUMP,
+    /*  4 */ GELDB_ROLL_BACK,
+    /*  5 */ GELDB_READY,
+    /*  6 */ GELDB_BLOCK,
+    /*  7 */ GELDB_SLASH,
+    /*  8 */ GELDB_ADVANCE,
+    /*  9 */ GELDB_PIVOT,
+    /* 10 */ GELDB_CIRCLE,
+    /* 11 */ GELDB_UNUSED,
+    /* 12 */ GELDB_SPIN_ATTACK,
+    /* 13 */ GELDB_SIDESTEP,
+    /* 14 */ GELDB_ROLL_FORWARD,
+    /* 15 */ GELDB_STUNNED,
+    /* 16 */ GELDB_SPIN_DODGE
+} EnGeldBActionState;
+
+typedef enum {
+    /* 0 */ GELDB_DAMAGE_NORMAL,
+    /* 1 */ GELDB_DAMAGE_STUN,
+    /* 6 */ GELDB_DAMAGE_UNK6 = 6,
+    /* D */ GELDB_DAMAGE_UNKD = 0xD,
+    /* E */ GELDB_DAMAGE_UNKE,
+    /* F */ GELDB_DAMAGE_FREEZE
```

Comment:
decimal is prob a good idea. or if you want to keep them hex, prefix the offsets with 0x

Reply:
The reason for hex is that they have a one-to-one correspondence with the upper hex of the damage table entries.

Reply:
alright, then yea the offsets should have 0x